models/SequenceVerificationDataCentric/models/SequenceVerificationDataCentric/SVDCdomain/AESequenceDC/SVDCclasses/SequencedAuditEvent/SequencedAuditEvent.masl

```diff
@@ -105,10 +101,9 @@ begin
     end if;
   else
       // Log error: The audit event id received has been seen before
-	  logMessage := "AESequenceDC::SequencedAuditEvent.AddEventForNewJob : The audit event has been seen before for jobId = " & jobId & " with Job Name = " & jobName & ", Initial audit event = " & aeType;
-	  Logger::log(Logger::Debug, "AESequenceDC", logMessage);
-      eventContent := "JobId = " & jobId & " with Job Name = " & jobName & " : FailureReason = The audit event has been seen before"  & " : EventId = " & eventId & " : EventType = " & aeType;
-	  Reporting~>reportEvent(Logger::Error, "svdc_job_failed", eventContent);  
+      // TODO:  CDS - Does the job (state machine) get failed?
+      logMessage := "svdc_job_failed : JobId = " & jobId & " with Job Name = " & jobName & " : FailureReason = The audit event has been seen before"  & " : EventId = " & eventId & " : EventType = " & aeType;
+      Logger::log(Logger::Warning, "pv.sv.sequencedauditevent.addeventfornewjob", logMessage);
 
```

Comment:
Should this be pv.sv.sequenceauditevent.addeventfornewhappyjob (to be consistent with pv.sv.sequenceauditevent.addeventfornewunhappyjob below)?